components/org.wso2.carbon.identity.data.publisher.application.authentication/pom.xml

```diff
@@ -75,22 +80,39 @@
                             org.wso2.carbon.identity.data.publisher.application.authentication.internal,
                         </Private-Package>
                         <Import-Package>
-                            !org.wso2.carbon.identity.data.publisher.application.authentication.*,
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component.*; version="${osgi.service.component.imp.pkg.version.range}",
                             org.wso2.carbon; version="${carbon.kernel.package.import.version.range}",
-                            org.apache.commons.logging; version="${commons-logging.osgi.version.range}",
                             org.wso2.carbon.identity.application.authentication.framework.*;version="${carbon.identity.framework.package.import.version.range}",
-                            org.wso2.carbon.databridge.commons; version="${carbon.analytics-common.version}",
-                            *;resolution:=optional
+
+                            org.apache.commons.lang,
```

Comment:
Can we add import version ranges if possible ? (To avoid future problems)

Reply:
import version ranges were added.